packages/scripts/webapp/preset/README.md

```diff
@@ -277,7 +277,7 @@ To use the eslint configuration in your IDE
 
 ```json
 {
-  "extends": "@talend/scritps/preset/config/.eslintrc"
```

Comment:
Just to note, I was aware of the typo and path diff also, and it doesn't solve the issue only changing that